docs/user-guide/kubectl-overview.md

```diff
@@ -33,160 +33,271 @@ Documentation for other releases can be found at
 
 # kubectl overview
 
-**Table of Contents**
+Use this overview of the `kubectl` command line interface to help you start running commands against Kubernetes clusters. This overview quickly covers `kubectl` syntax, describes the command operations, and provides common examples. For details about each command, including all the supported flags, see the [kubectl](kubectl/kubectl.md) reference documentation.
 
-- [kubectl overview](#kubectl-overview)
-  - [Overview](#overview)
-  - [Common Operations](#common-operations)
-  - [Kubectl Operations](#kubectl-operations)
-  - [Resource Types](#resource-types)
+**Table of Contents**
 
-This overview is intended for anyone who wants to use `kubectl` command line tool to interact with Kubernetes cluster. Please remember that it is built for quick started with `kubectl`; for complete and detailed information, please refer to [kubectl](kubectl/kubectl.md).
+ * [Syntax](#syntax)
+ * [Operations](#operations)
+ * [Resource types](#resource-types)
+ * [Output options](#output-options)
+ * [Examples: Common operations](#examples-common-operations)
 
-TODO: auto-generate this file to stay up with `kubectl` changes. Please see [#14177](https://github.com/kubernetes/kubernetes/pull/14177).
+TODO: Auto-generate this file to ensure it's always in sync with any `kubectl` changes, see [#14177](https://github.com/kubernetes/kubernetes/pull/14177).
 
-## Overview
+## Syntax
 
-`kubectl` controls the Kubernetes cluster manager. The synopsis is:
+Use the following syntax to run `kubectl` commands from your terminal window:
 
 ```
 kubectl [command] [TYPE] [NAME] [flags]
 ```
 
-This specifies:
+where `command`, `TYPE`, `NAME`, and `flags` are:
+
+* `command`: Specifies the operation that you want to perform on one or more resources, for example `create`, `get`, `describe`, `delete`.
+* `TYPE`: Specifies the [resource type](#resource-types). Resource types are case-sensitive and both the singular and plural forms are valid. For example, the following two commands produce the same output:
+
+   ```
+    $ kubectl get pod pod1
+    $ kubectl get pods pod1
+   ```
```

Comment:
*Revised to:*

Resource types are case-sensitive and you can specify the singular, plural, or abbreviated forms. For example, the following commands produce the same output:

   ```
    $ kubectl get pod pod1
    $ kubectl get pods pod1
    $ kubectl get po pod1
   ```